modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -102,6 +141,104 @@ neo4j@neo4j> SHOW SERVERS YIELD *;
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+
+The table of results shows information about the servers:
+
+[options="header", width="100%", cols="2a,4,2m,1,1"]
+|===
+| Column
+| Description
+| Type
+| Default output
+| Full output
+
+| name
+| Name of the server.
+| STRING
+| {check-mark}
+| {check-mark}
+
+| serverId
+| Id of the server.
+| STRING
+|
+| {check-mark}
+
+| address
+| Bolt address of the server (if enabled).
+| STRING
+| {check-mark}
+| {check-mark}
+
+| httpAddress
+| Http address of the server (if enabled).
+| STRING
+|
+| {check-mark}
+
+| httpsAddress
+| Https address of the server (if enabled).
+| STRING
+|
+| {check-mark}
+
+| state
+| Information of the state of the server: `free`, `enabled`, `deallocating`, or `dropped`.
+| STRING
+| {check-mark}
+| {check-mark}
+
+| health
+| The availability of the server: `available` or `unavailable`.
+| STRING
+| {check-mark}
+| {check-mark}
+
+| hosting
+| A list of databases currently hosted on the server.
```

Comment:
This maybe we should rephrase somehow. "hosted" is not completely true. If a database is just starting or stopping or is in a quarantine on that server, it will show up here. In all these cases one could argue that the database is *not* hosted - since that database is not available. Normally I go with the phrase "the server reports a state for that database", which is very technical. So I don't really have a good idea what to write, but this sentence alone will lead (or already lead) to misunderstandings.

Reply:
Since the column returned is `hosted` I don't think that the description in the docs is going to help possible confusion. So I will leave as is for the scope of this PR.